src/app/forms/models/testTypeId.enum.ts

```diff
@@ -188,9 +188,9 @@ export const SPECIALIST_TEST_TYPE_IDS: string[] = [
   '185'
 ];
 
+export const TEST_TYPES_GROUP1_DESK_BASED_TEST: string[] = ['418'];
 
-export const TEST_TYPES_GROUP1_DESK_BASED_TEST: string[] = ['417', '418'];
```

Comment:
Did you mean to delete `417`?

Reply:
Yeah I'm unsure how we're grouping these because the vehicle section is completely different between group 1 psv and group 1 trl